test/Migrator/iuo_any_coercion.swift

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: objc_interop
+// RUN: %empty-directory(%t) && %target-swift-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/Inputs/API.json -emit-migrated-file-path %t/iuo_any_coercion.swift.result -emit-remap-file-path %t/iuo_any_coercion.swift.remap -o /dev/null
+// RUN: diff -u %S/iuo_any_coercion.swift.expected %t/iuo_any_coercion.swift.result
+
+let x: Int! = 1
+let _: Any = x
+
+let y: Int? = 1
+let _: Any = y
```

Comment:
Please add tests for member references, function calls, subscripts, as well a test where the code already has 'as Any' (and thus we wouldn't expect the note, nor the migration).